src/components/sprite-info/sprite-info.css

```diff
@@ -79,3 +79,9 @@
     user-select: none;
     outline: none;
 }
+
+@media only screen and (max-width: 1096px) {
+    .non-essential {
+        display: none;
+    }
+}
```

Comment:
From offline: Let's consolidate to one method, and I think using `MediaQuery` will be the most flexible/compatible.